src/services/Api.php

```diff
@@ -58,4 +58,36 @@ public function getClient()
     {
         return $this->_client;
     }
+
+    /**
+     * @param $order
+     * @return mixed
+     */
+    public function createOrder($order)
+    {
+        $taxOrderData = [
+          'transaction_id' => $order->id,
+          'transaction_date' => $order->datePaid->format('Y/m/d'),
+          'to_country' => $order->shippingAddress->country->iso,
+          'to_zip' => $order->shippingAddress->zipCode,
+          'to_state' => $order->shippingAddress->state ? $order->shippingAddress->state->abbreviation : $order->shippingAddress->stateName,
+          'to_city' => $order->shippingAddress->city,
+          'to_street' => $order->shippingAddress->address1,
+          'amount' => $order->total,
```

Comment:
@aryehischechter Amount is total? And line items is an empty array?

Reply:
Total amount of the order with shipping, excluding sales tax in dollars.
so we need total - tax. i'll update it.